src/main/resources/l10n/Menu_de.properties

```diff
@@ -87,7 +87,7 @@ Import_into_current_database=Importieren_in_aktuelle_&Datenbank
 
 Switch_to_%0_mode=In_den_%0-Modus_wechseln
 Push_entries_to_external_application_(%0)=&Eintr\u00e4ge_in_externe_Anwendung_einf\u00fcgen_(%0)
-Pull_changes_from_shared_database=Entfernte_Änderungen_beziehen
+Pull_changes_from_shared_database=\u00c4nderungen_der_geteilten_Datenbank_beziehen
```

Comment:
Do not use `\u00c4`, use the Umlaut directly. We have UTF-8 and this works perfectly.